examples/stm32/nucleo-f429zi-baremetal/hal.h

```diff
@@ -146,22 +92,28 @@ static inline void irq_exti_attach(uint16_t pin) {
   EXTI->RTSR |= BIT(n);
   EXTI->FTSR |= BIT(n);
   int irqvec = n < 5 ? 6 + n : n < 10 ? 23 : 40;  // IRQ vector index, 10.1.2
-  nvic_set_prio(irqvec, 3);
-  nvic_enable_irq(irqvec);
+  NVIC_SetPriority(irqvec, 3);
+  NVIC_EnableIRQ(irqvec);
 }
 
-struct uart {
+#if 0
+USART_TypeDef {
   volatile uint32_t SR, DR, BRR, CR1, CR2, CR3, GTPR;
 };
-#define UART1 ((struct uart *) 0x40011000)
-#define UART2 ((struct uart *) 0x40004400)
-#define UART3 ((struct uart *) 0x40004800)
+#define UART1 ((USART_TypeDef *) 0x40011000)
+#define UART2 ((USART_TypeDef *) 0x40004400)
+#define UART3 ((USART_TypeDef *) 0x40004800)
+#endif
```

Comment:
Why keeping these ? (and others)

Reply:
Good point! let me clean that up in my next PR which is coming ...